src/goto-programs/remove_exceptions.cpp

```diff
@@ -70,7 +70,7 @@ Date:   December 2016
 /// and catch_pop instructions indicate how they should be checked to dispatch
 /// the right exception type to the right catch block, and landingpad
 /// instructions copy back to an ordinary local variable (or other expression)
-/// and set #exception_value back to null, indicating the exception has been
+/// and set \#exception_value back to null, indicating the exception has been
```

Comment:
According to stack overflow at least that doesn't work, it still considers it a link and the backslash escape doesn't work. Apparently \c \# can work though.